### PR TITLE
Fix css build

### PR DESCRIPTION
**Pull request by urmauur**
## Description
- Fixing css build #145 
- Add boostrap from npm
- Add gulp rename
- Minify css build

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (security config, docs update, etc.)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,11 +30,13 @@
   "dependencies": {
     "axios": "^0.19.0",
     "body-parser": "^1.15.1",
+    "bootstrap": "^4.5.2",
```
Comment (wildan3105) [inline package.json:33]: why do we need install bootstrap? where do we use it?

File: views/partials/head.handlebars [hunk 2]
```diff
@@ -22,9 +22,8 @@
   <title>Github Language Checker {{ title }}</title>
 
   <!-- Bootstrap core CSS -->
-  <link href="public/css/bootstrap.min.css" rel="stylesheet">
```
Comment (urmauur) [inline views/partials/head.handlebars:25]: @wildan3105 here your bootstrap

File: package.json (hunk 1 above)
Comment (urmauur) [reply package.json:33]: @wildan3105 you add static file bootstrap I think, and the file still beta bootstrap 4, so I think it's good to install bootstrap from npm so you can update anytime, you use bootstrap on field input and button group for search, see the screenshot

Screen Shot 2020-10-02 at 16 36 34

File: views/partials/head.handlebars (hunk 2 above)
Comment (wildan3105) [reply views/partials/head.handlebars:25]: ok got it

**Review by wildan3105 — APPROVED** [review]
LGTM!

**Comment by wildan3105** [comment]
One last thing, mind to bump version to `2.0.14` in package.json @urmauur ?

**Comment by urmauur** [comment]
> One last thing, mind to bump version to `2.0.14` in package.json @urmauur ?

sure

**Comment by wildan3105** [comment]
Merging this, thank you for the contribution @urmauur !

**Review by wildan3105 — APPROVED** [review]

**Comment by urmauur** [comment]
@wildan3105 also this PR for `hacktoberfest-accepted` 🙏🏻

**Pull request merged**